Dockerfile

```diff
@@ -190,7 +190,7 @@ ENV PATH   $PATH:/usr/lib/go/bin
 RUN echo " ... installing api-gateway-config-supervisor  ... " \
     && echo "http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
     && apk update \
-    && apk add make git go \
+    && apk add gcc make git go \
```

Comment:
Can we delete GCC after finishing the installation ? See the lines bellow with apk del ...